### ISSUE TITLE
Head assignment for bottom data not working

### ISSUE THREAD

**Issue by nseidle** [sparkfunX/Desktop-PickAndPlace-CHMT36VA#12]
I ran my first double sided board and the head assignment for the feeders isn't getting through.

**Comment by berniwa** [comment]
Seems weird, worked fine here.
Could you try my latest version?  https://github.com/berniwa/Desktop-PickAndPlace-CHMT36VA/tree/master/Eagle-Conversion
(Now supports tray handling, but I still need to do some more tweeking so I didn't push it)

**Comment by nseidle** [comment]
Sorry, this is not an issue.

When I looked at the feeder info for a job I was doing I saw the feeder head was 1:

![image](https://user-images.githubusercontent.com/117102/38164545-a0357186-34c3-11e8-9067-bdeec8571c97.png)

But this is a limitation of the Charm High software: the dpv file does not contain head number in the feeder record. Head data is only stored in the component data as shown below:

![image](https://user-images.githubusercontent.com/117102/38164561-c7373364-34c3-11e8-8acb-1dd656ed12f2.png)

Head 2 was correctly coming through. I was just confused because I was looking at the feeder record instead of the component record.
